app/views/main.hbs

```diff
@@ -105,9 +116,10 @@
 
     {{! Mount point for React }}
     <div id="react-app" class="app"></div>
-
     <script src="https://platform.twitter.com/widgets.js" async></script>
     <script src="/assets/main.{{cacheTimestamp}}.js"></script>
+    <script src="https://cdn.coil.com/coil-oauth-wm.v7.beta.js">
+    </script> 
```

Comment:
Let's remove the newline (and any whitespace) between opening and closing script tags for consistency.